code/modules/hydroponics/trays/tray_soil.dm

```diff
@@ -45,7 +45,7 @@
 
 /obj/machinery/portable_atmospherics/hydroponics/soil/invisible/New(var/newloc,var/datum/seed/newseed)
 	..()
-	seed = newseed
+	seed = new newseed
```

Comment:
This doesn't make sense. It's already given a datum object, why are you cloning it? Unless it doesn't work?